shells/shells.go

```diff
@@ -0,0 +1,50 @@
+package shells
+
+import (
+	"github.com/nopecmd/nope/models"
+	"github.com/rogpeppe/rog-go/reverse"
+	"os"
+)
+
+var CurrentShell models.Shell
+
+func init() {
+	CurrentShell = models.Shell{Name: "fish", GetLastCmd: getLastLineFish}
+}
+
+func getLastLineBash() string {
+	var fname = os.Getenv("HOME") + "/.bash_history"
+	file, err := os.Open(fname)
+
+	if err != nil {
+		panic(err)
+	}
+	defer file.Close()
+
+	scanner := reverse.NewScanner(file)
+	scanner.Scan()
+	return scanner.Text()
+}
+
+func getLastLineFish() string {
+	var fname = os.Getenv("HOME") + "/.config/fish/fish_history"
```

Comment:
also same thing for `/.bash_history`, and we can make a function to have a cleaner abstraction for this, something like:

```go
func getFileNameFromHome(path string) string {
    return os.Getenv("HOME") + path
}
```